CHANGELOG.md

```diff
@@ -2,9 +2,19 @@
 
 Friendly reminder, we have a [bug bounty program](https://hackerone.com/cosmos).
 
+## v0.34.20-rc1
```

Comment:
Did the lazy log string thing get added in between?

Reply:
I didn't see it in the git logs, I think possibly not backported yet?